pkg/constants/envvar.go

```diff
@@ -0,0 +1,16 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package constants
+
+const (
+	DDAPIKey      = "DD_API_KEY"
+	DDAppKey      = "DD_APP_KEY"
+	DDddURL       = "DD_DD_URL"
+	DDURL         = "DD_URL"
+	DDSite        = "DD_SITE"
+	DDLogLevel    = "DD_LOG_LEVEL"
```

Comment:
nit: Could possibly move `DDLogLevel` over to `internal/controller/datadogagent/override/envvar.go` as it's only used in `internal/controller/datadogagent/override/` multiple times **and once** in `internal/controller/testutils` where it could be simply replaced as a string `DD_LOG_LEVEL`